.deps/resolved/linux-aarch64_3.12.txt

```diff
@@ -64,7 +64,8 @@ ply @ https://agent-int-packages.datadoghq.com/external/ply/ply-3.11-py2.py3-non
 prometheus-client @ https://agent-int-packages.datadoghq.com/external/prometheus-client/prometheus_client-0.22.1-py3-none-any.whl#sha256=cca895342e308174341b2cbf99a56bef291fbc0ef7b9e5412a0f26d653ba7094
 protobuf @ https://agent-int-packages.datadoghq.com/external/protobuf/protobuf-6.31.1-cp39-abi3-manylinux2014_aarch64.whl#sha256=a40fc12b84c154884d7d4c4ebd675d5b3b5283e155f324049ae396b95ddebc39
 psutil @ https://agent-int-packages.datadoghq.com/external/psutil/psutil-6.0.0-cp36-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl#sha256=e2e8d0054fc88153ca0544f5c4d554d42e33df2e009c4ff42284ac9ebdef4132
-psycopg2-binary @ https://agent-int-packages.datadoghq.com/external/psycopg2-binary/psycopg2_binary-2.9.9-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl#sha256=8f8544b092a29a6ddd72f3556a9fcf249ec412e10ad28be6a0c0d948924f2212
+psycopg @ https://agent-int-packages.datadoghq.com/external/psycopg/psycopg-3.2.9-py3-none-any.whl#sha256=01a8dadccdaac2123c916208c96e06631641c0566b22005493f09663c7a8d3b6
+psycopg-binary @ https://agent-int-packages.datadoghq.com/external/psycopg-binary/psycopg_binary-3.2.9-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl#sha256=96a551e4683f1c307cfc3d9a05fec62c00a7264f320c9962a67a543e3ce0d8ff
```

Comment:
That, I'm not sure yet. @alopezz?

Reply:
This is expected given how psycopg binary is distributed: https://github.com/psycopg/psycopg/blob/dfce629f2a8e867256cf35383cfa0a66c93ae8a8/psycopg/setup.cfg#L66-L67

When pip installing `psycopg[binary]`, that installs `psycopg` with the optional `binary` dependency set which includes `psycopg-binary`.